.github/workflows/test-python.yml

```diff
@@ -0,0 +1,63 @@
+---
+name: test-python
+
+on:
+  push:
+    branches:
+      - main
+      - renovate/**
+  pull_request:
+    branches:
+      - main
+  workflow_dispatch:
+
+jobs:
+  build:
+
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        include:
+          - os: ubuntu-latest
+            python-version: "3.9"
+          - os: ubuntu-latest
+            python-version: "3.10"
+          - os: ubuntu-latest
+            python-version: "3.11"
+          - os: ubuntu-latest
+            python-version: "3.12"
+          - os: ubuntu-latest
+            python-version: "3.13"
+          - os: ubuntu-latest
+            python-version: "pypy3.9"
+          - os: ubuntu-latest
+            python-version: "pypy3.10"
+          - os: windows-latest
+            python-version: "3.13"
+          - os: macos-latest
+            python-version: "3.13"
+
+    steps:
+      - uses: actions/checkout@v4
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v5
+        with:
+          python-version: ${{ matrix.python-version }}
+      - name: Generate code
```

Comment:
This shouldn't be needed as this should be part of the regular code generation and checkin process

Reply:
It's needed for test execution later: We have to know if the model is fine for serialization/deserialization

Reply:
Sorry I think you're misunderstanding. The code "itself" is checked in (See this PR below for the code `_messages.py` I think is the single file with all messages.

As part of your regular development flow you can develop new models and/or new facilities for enum's e.t.c. - Then once you run `make generate` it will generate all your new code. This is the same flow as every other language

This code (Running `make generate`), would therefore "re"generate the code on each and every PR which isn't something we would want to do, as it would create a rather large overhead. So you should just run `make generate` whenever you want to update your messages

Reply:
Fixed